pages/games/[gameType].tsx

```diff
@@ -18,20 +18,7 @@ import {
 import styles from 'pages/games/gameType.module.scss';
 import { TicTacToe, PlayerType as TicTacToePlayerType } from "components/Games/Tic-Tac-Toe";
 import { TicTacToeState } from "components/Games/Tic-Tac-Toe/types";
-import gameApi, {
-  isValidGameMove,
-  isValidSignedMove,
-  getArbiter,
-  getSigner,
-  getRulesContract,
-  finishGame,
-  disputeMove,
-  initTimeout,
-  resolveTimeout,
-  finalizeTimeout,
-  FinishedGameState,
-  RunDisputeState,
-} from "../../gameApi";
+import gameApi, { isValidSignedMove, getArbiter, getSigner, getRulesContract, finishGame, disputeMove, initTimeout, resolveTimeout, finalizeTimeout, FinishedGameState, RunDisputeState, getArbiterRead, isValidGameMove } from "../../gameApi";
```

Comment:
@poludnev  please pick your formatting. Last time you reformatted this from flat to column view
https://github.com/chainhackers/gamejutsu-demos/commit/e2bb72b6c0c294a81ccafdaf5b9de4cf990518f0